libs/shared/console-shared/src/lib/buttons-actions/ui/application-buttons-actions/application-buttons-actions.tsx

```diff
@@ -271,6 +272,8 @@ export function ApplicationButtonsActions(props: ApplicationButtonsActionsProps)
     }
 
     setButtonStatusActions([{ items: topItems }, { items: bottomItems }])
+    // loop with actionCancelEnvironment if added
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why is it in a `useEffect` in the first place?
To me, all this function is about setting `buttonStatusActions` and this do not need `useEffect`.
Eventually `useMemo` but definitely not `useEffect`